### PR TITLE
Update snap to 3.2.0.742 and event-collector for logsv2 (AO-16825)

### PR DESCRIPTION
**Pull request by d-maslyk**
* Update SWI SNAP binary in Dockerfile to latest -  3.2.0.742
* Update event-collector object to utilize logs plugin in v2 version (non backward compatible change)
* Bump major version in Makefiles (due introducing non backward compatible changes in k8s object)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,92 +241,22 @@ If you use `SWISNAP_ENABLE_<plugin_name>` set to `true`, then keep in mind that
...
-        connect_timeout: "30s"
-
-        write_timeout: "30s"
-
-        files: |
-          /var/log/SolarWinds/Snap/events.log
-
-        exclude_patterns: |
-          .*self-skip-logs-collector.*
+Version 22 of Kubernetes collector allows you to collect cluster events and push them to Loggly using logs collector under the hood. To utilise this functionality there is a need to create corresponding configmaps in your cluster, with proper plugins configration. Those files can be found in  [Event collector configs](examples/event-collector-configs). To enable event collection in your deployment, follow below steps:
```
Comment (tomekkolo) [inline README.md:244]: please add a link to documentation page: https://documentation.solarwinds.com/en/Success_Center/appoptics/Content/kb/host_infrastructure/host_agent/kubernetes_ha.htm#integrating-kubernetes-cluster-events-collection-with-loggly

File: deploy/base/events-collector/swisnap-agent-deployment-event-collector.yaml [hunk 2]
```diff
@@ -23,17 +23,17 @@ spec:
...
               subPath: kubernetes.yaml
             - name: tasks-aokubernetes
               mountPath: /opt/SolarWinds/Snap/etc/tasks.d/task-aokubernetes.yaml
               subPath: task-aokubernetes.yaml
-            - name: tasks-logs
-              mountPath: /opt/SolarWinds/Snap/etc/tasks.d/task-logs.yaml
-              subPath: task-logs.yaml
+            - name: tasks-autoload-logs
+              mountPath: /opt/SolarWinds/Snap/etc/tasks-autoload.d/task-logs-k8s.yaml
+              subPath: task-logs-k8s.yaml
```
Comment (tomekkolo) [inline deploy/base/events-collector/swisnap-agent-deployment-event-collector.yaml:36]: why not tasks-logs-v2.yaml ?

Comment (tomekkolo) [reply deploy/base/events-collector/swisnap-agent-deployment-event-collector.yaml:36]: yeah, logs-k8s makes more sense as there might be more log tasks.

File: README.md (hunk 1 above)
Comment (d-maslyk) [reply README.md:244]: Link added

**Pull request merged**